### PR TITLE
Added BH1750FVI Binding

### LINKED ISSUE #181: Add bindings for BH1750 (Light Sensor)

### PR DESCRIPTION
**Pull request by ZhangGaoxing**
fix #181 

@joperezr

### CODE REVIEW

File: src/devices/Bh1750fvi/Bh1750fvi.cs [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    /// Ambient Light Sensor BH1750FVI
+    /// </summary>
+    public class Bh1750fvi : IDisposable
+    {
+        private const byte DefaultLightTransmittance = 0b_0100_0101;
+
+        private I2cDevice _sensor;
+
+        private double _lightTransmittance;
+        /// <summary>
```
Comment (krwq) [inline src/devices/Bh1750fvi/Bh1750fvi.cs]: enter before comment

File: src/devices/Bh1750fvi/Bh1750fvi.cs [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        /// BH1750FVI Light Transmittance, from 27.20% to 222.50%
+        /// </summary>
+        public double LightTransmittance
+        {
+            get => _lightTransmittance;
```
Comment (krwq) [inline src/devices/Bh1750fvi/Bh1750fvi.cs]: should we read this from register?

File: src/devices/Bh1750fvi/Bh1750fvi.cs [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+            LightTransmittance = lightTransmittance;
+            MeasuringMode = measuringMode;
+        }
+
+        /// <summary>
+        /// Cleanup
+        /// </summary>
+        public void Dispose()
+        {
+            _sensor?.Dispose();
```
Comment (krwq) [inline src/devices/Bh1750fvi/Bh1750fvi.cs]: `_sensor = null;`

File: src/devices/Bh1750fvi/Bh1750fvi.cs [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+                _sensor.WriteByte((byte)Command.PowerOn);
+
+            Span<byte> readBuff = stackalloc byte[2];
+
+            _sensor.WriteByte((byte)MeasuringMode);
+            _sensor.Read(readBuff);
+
+            ushort raw = BinaryPrimitives.ReadUInt16BigEndian(readBuff);
+
+            if (MeasuringMode == MeasuringMode.ContinuouslyHighResolutionMode2 || MeasuringMode == MeasuringMode.OneTimeHighResolutionMode2)
```
Comment (krwq) [inline src/devices/Bh1750fvi/Bh1750fvi.cs]: would it make sense to simplify this to:

```csharp
double result = raw / (1.2 * _lightTransmittance);

if (... current condition ...)
{
   result *= 2;
}

return result;
```

File: src/devices/Bh1750fvi/Bh1750fvi.cs (hunk 2 above)
Comment (ZhangGaoxing) [reply src/devices/Bh1750fvi/Bh1750fvi.cs]: The sensor doesn't seem to provide a register address table, it just gives the setup command...

File: src/devices/Bh1750fvi/Bh1750fvi.cs (hunk 3 above)
Comment (ZhangGaoxing) [reply src/devices/Bh1750fvi/Bh1750fvi.cs]: 😅

**Pull request merged**